### PR TITLE
Loading Mask

### PR DESCRIPTION
**Pull request by sri2k1us**
- Use Loading Mask from `de-components` instead of default `CircularProgress`.
- Remove `jss` styles.
- Remove  `jss` deps.

Note: This branch is on top of #411. So only last 2 commits needs to reviewed.

### CODE REVIEW

File: react-components/src/communities/view/EditCommunity.js [hunk 1]
```diff
@@ -469,16 +469,11 @@ class EditCommunity extends Component {
...
-                                        classes={{ root: classes.loading }}
-                                        thickness={7}
-                                    />
-                                )}
-                                {getMessage("confirmDeleteCommunity")}
-                            </div>
+                            <LoadingMask loading={loading}>
+                                <div>
+                                    {getMessage("confirmDeleteCommunity")}
+                                </div>
```
Comment (ash3rz) [inline react-components/src/communities/view/EditCommunity.js]: Minor, but are these added `<div>` tags needed?

Comment (sri2k1us) [reply react-components/src/communities/view/EditCommunity.js]: I actually moved the `div` tag that was outside the `CircularProgress` into the `LoadingMask`. But ya I can check to see if this `div` can be removed.

File: react-components/src/metadata/EditMetadata.js [hunk 2]
```diff
@@ -211,175 +212,176 @@ class EditMetadata extends Component {
...
-                <DialogContent>
-                    {loading ? (
-                        <CircularProgress
-                            className={classes.loadingStyle}
-                            size={50}
-                            thickness={4}
-                        />
-                    ) : (
+                    <DialogContent>
+                        (
```
[the remark this answers is not in the document]
Comment (sri2k1us) [reply react-components/src/metadata/EditMetadata.js]: Man you got some 👀 . Good Catch!

[the remark this answers is not in the document]
Comment (psarando) [reply react-components/src/metadata/EditMetadata.js]: 😆 Actually I only noticed it when I launched Storybook as part of testing my `webpack.config` changes.

**Comment by sri2k1us** [comment]
Thanks for the reviews.

**Pull request merged**